### PR TITLE
feat: app and stats store

### PR DESCRIPTION
**Pull request by manuel-calavera**
stores impl. does not affect existent app.

### CODE REVIEW

File: src/store/stats.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+    if (newTransactionCount < transactionCount / 2) {
+      this.clusterRestart();
+      return;
+    }
+
+    this.stats.totalSupply = yield this.connection.getTotalSupply();
+    const newMined = yield this.replicator.adjustedReplicatorBalance();
+    if (newMined > depositMinimumLamports) {
+      if (isValidPublicKey(AppStore.depositPublicKey)) {
```
[the remark this answers is not in the document]
Comment (manuel-calavera) [reply src/store/stats.js]: removed

File: src/store/app.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
+import LocalStore from 'electron-store';
+import { action, observable } from 'mobx';
+
+export const localStore = new LocalStore();
```
Comment (mvines) [inline src/store/app.js]: Why `localStore` exported?

File: src/store/stats.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
+import LocalStore from 'electron-store';
+import { Connection, PublicKey } from '@solana/web3.js';
+import { action, observable, flow } from 'mobx';
+import url from '../url';
+import Replicator from '../replicator';
+import AppStore from './app';
+
+const DEFAULT_LAMPORTS = 10000;
+export const localStore = new LocalStore();
```
Comment (mvines) [inline src/store/stats.js]: Why `localStore` exported?

File: src/store/stats.js [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+  }
+
+  @observable stats = {
+    transactionCount: 0,
+    totalMined: localStore.get('totalMined', 0),
+    newMined: 0,
+    totalSupply: 0,
+    depositMinimumLamports: localStore.get(
+      'depositMinimumLamports',
+      DEFAULT_LAMPORTS
```
Comment (mvines) [inline src/store/stats.js]: IMO just inline the `10000` here.  That `DEFAULT_LAMPORTS` constant isn't doing much

File: src/store/app.js (hunk 2 above)
Comment (manuel-calavera) [reply src/store/app.js]: not needed

File: src/store/stats.js (hunk 3 above)
Comment (manuel-calavera) [reply src/store/stats.js]: export is not needed. will rm.

File: src/store/stats.js (hunk 4 above)
Comment (manuel-calavera) [reply src/store/stats.js]: no magic numbers :) next pr i will move this to other constants to constants.js file. stay tuned!

**Pull request merged**